input/fsh/value_sets.fsh

```diff
@@ -146,3 +147,17 @@ Other value sets may be added in the future."
 * SCT#720735008 "Presumptive positive (qualifier value)"
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////
+
+ValueSet:    IdentityAssuranceLevelValueSet
+Id:          identity-assurance-level-value-set
+Title:       "Identity assurance level value set"
+Description: "Code representing identity assurance level, based on NIST 800-63-3"
+
+* LOA#IAL1 "Name and birth date were self-asserted."
+* LOA#IAL1.2 "Strong and/or Fair identity evidence other than a US state-issued photo ID or nationally-issued photo ID was used to verify name and birth date."
+* LOA#IAL1.4 "A US state-issued photo ID or nationally-issued photo ID was used to verify name and birth date."
+* LOA#IAL1.5 "An unspecified ID was used to verify name and birth date."
```

Comment:
unspecified doesn't seem like it should be *stronger* than "A US state-issued photo ID"... so I'm not sure about picking a value of 1.5 (>1.4). I wonder if we could do the following (with guidance that issuers should pick the highest level that applies):

```suggestion
* LOA#IAL1 "Name and birth date were self-asserted."
* LOA#IAL1.2 "An unspecified ID was used to verify name and birth date."
* LOA#IAL1.4 "A US state-issued photo ID or nationally-issued photo ID was used to verify name and birth date."
```